code/modules/medical/genetics/bioEffects/harmful.dm

```diff
@@ -588,6 +588,30 @@
 		if (prob(emote_prob))
 			L.emote(emote_type)
 
+/datum/bioEffect/colorblindness
+	name = "Protanopia"
+	desc = "Selectively inhibits the L-cones in the subject's eyes, causing red-green colorblindness."
+	id = "protanopia"
+	effectType = effectTypeDisability
+	isBad = 1
+	msgGain = "Everything starts looking a lot more yellow."
+	msgLose = "You notice a few extra colors."
+	probability = 99
+	icon_state  = "bad"
+	var/list/protanopia_matrix = list(MATRIX_PROTANOPIA)
+
+	OnAdd()
+		src.removed = 0
+		APPLY_MOB_PROPERTY(owner, PROP_PROTANOPIA, src)
+		owner.client?.color = list(protanopia_matrix)
```

Comment:
Remove the list () call here too.

Reply:
Done. Dunno why it works now doing that, cus before it just made everything blue.